client/src/panels/panel_declarations/finances/transfer_payments/tp_by_region.yaml

```diff
@@ -6,39 +6,29 @@ tp_by_region_title:
 
-  fr: |
-    #TODO
-    {{#if show_per_capita}}  
-    En **{{pa_last_year}}**, **{{subj_name subject}}** a versé **{{total_sum}}** en paiements de transfert. Les provinces bénéficiaires des paiements de transfert au-dessus 100 000 $ sont présentés ci-dessous, un montant divisé par la population de la région sélectionnée. Les bénéficiaires de **{{largest_prov}}** ont reçu la plus grande part divisée de **{{largest_per_capita}}** par habitant, soit un total de **{{largest_amount}}**.
-    {{else}}
-    En **{{pa_last_year}}**, **{{subj_name subject}}** a versé **{{total_sum}}** en paiements de transfert. Les provinces bénéficiaires des paiements de transfert au-dessus 100 000 $ sont présentés ci-dessous. Les bénéficiaires de **{{largest_prov}}** ont reçu la plus grande part totale de **{{largest_amount}}** ({{percent_of_total}}).
+    {{#if show_per_capita_data}}
+      When accounting for population statistics, the region with the largest per capita transfer payment value in **{{pa_last_year}}** was **{{largest_per_capita_prov}}**, at **{{fmt_compact1_written largest_per_capita_value}} per capita**.
+      {{#if compare_per_capita_to_largest_total}}
+        For comparison, the region receiving the largest total value, **{{largest_total_prov}}**, received **{{fmt_compact1_written largest_total_per_capita_value}} per capita**.
+      {{/if}}
     {{/if}}
-    
-    Survolez ou tapez sur une province ou un territoire dans la carte ci-dessous pour voir le total des paiements de transfert au cours des cinq dernières années. Notez que l'échelle peut changer pour s'adapter aux données affichées.
+  fr: |
+    TODO
 
-show_tp:
-  en: Transfer Payments
-  fr: Les Paiements de Transfert
 
-show_tp_per_capita:
-  en: Tranfer Payments per Capita
-  fr: Les Paiements de Transfert par Habitant
+tp_by_region_graph_usage:
+  en: Hover over or tap a province or territory in the map below to see the total transfer payments over the last five years. Note that the scale may change to accommodate the data displayed.
+  fr: Survolez ou tapez sur une province ou un territoire dans la carte ci-dessous pour voir le total des paiements de transfert au cours des cinq dernières années. Notez que l'échelle peut changer pour s'adapter aux données affichées.
 
-tp_a11y_table_title:
-  en: Transfer Payments for Provinces
-  fr: Paiements de Transfert pour les Provinces
+transfer_payments:
+  en: Transfer Payments
+  fr: Paiements de transfert
 
-tp_pc_a11y_table_title:
-  en: Transfer Payments for Provinces per Capita
-  fr: Paiements de Transfert pour les Provinces par Habitant
+transfer_payments_per_capita:
+  en: Transfer Payments per Capita
+  fr: Paiements de transfert par habitant
```

Comment:
Technically the graph's 'regions', not provinces (provinces and territories, and also whatever category you'd put "Abroad" and "Not Available" in). Simpler to just not specify.

Reply:
Wording fix for yaml, makes sense
👍